webdriver-spec.html

```diff
@@ -486,6 +487,66 @@
 </section> <!-- /Conformance -->
 
 <section>
+<h2>Introduction</h2>
+
+<p><i>This section is non-normative</i>
+
+<h3>Design Notes</h3>
+
+<p>This specification attempts to follow a number of design goals:
+
+<section>
+<h4>Compatibility with existing Selenium WebDriver functionality</h4>
+
+<p>This spec is derived from the popular
+ <a href=https://seleniumhq.org>Selenium WebDriver</a> browser
+ automation framework. Selenium is a long-lived project, and due to
+ its age and breadth of use it has a wide range of expected
+ functionality. This specification uses these expectations to inform
+ its design. Where improvements or clarifications have been made, they
+ have been made with care to allow existing users Selenium WebDriver
+ to avoid unexpected breakages.
+</section>
+
+<section>
+<h4>Suitability for use end-user browser automation needs</h4>
+
+<p>It is expected that the largest group of users of this
+ specification are going to be software developers and testers writing
+ automated tests and other tooling, such as monitoring or load
+ testing, that relies on automating a browser. As such, care has been
+ taken to provide commands that simplify common tasks such
+ as <a href=#element-send-keys>typing into</a>
```

Comment:
remove the `href` and replace with `data-lt='Element Send Keys'` for send keys and `data-lt='Element Click'`